modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ModalCardSubscription/index.js

```diff
@@ -76,6 +85,13 @@ const ModalCardSubscription = ({
 	const totalCount =
 		subscriptionsTerms?.c?.accountSubscriptionTerms?.totalCount;
 
+	const removeProvisionedColumn = () => {
```

Comment:
I liked this function, but not the name, sounds like an action, but actually is returning columns

What you think about: `columnsWithoutProvisioned`

Or other name that put some emphasis in columns or a list of something

Reply:
cc: @aildsonf , @jairmedeiros

Reply:
Thanks @kevenleone for the suggestion, I strongly agree with that. It's fixed in commit ac0ed2f10fc439fff19c1fac3bbafc512380f13f

Reply:
Thank you